utils/kubernetes/env.go

```diff
@@ -36,3 +36,15 @@ func CreateOrReplaceEnv(container *v1.Container, name, value string) {
 		})
 	}
 }
+
+// AddIfNotPresent Adds and env variable to a container if not already present. Returns true if the variable didn't exist
+// and was added, false in any other case.
+func AddIfNotPresent(container *v1.Container, envVar v1.EnvVar) bool {
```

Comment:
```suggestion
func AddEnvIfNotPresent(container *v1.Container, envVar v1.EnvVar) bool {
```

We need to identify what this method is doing to our clients. Otherwise, we would have `kubeutil.AddIfNotPresent`. I know that we have the parameters, but even so looks better in the API doc.